xbmc/interfaces/json-rpc/JSONRPC.cpp

```diff
@@ -379,6 +379,15 @@ void CJSONRPCUtils::NotifyItemUpdated()
   CServiceBroker::GetGUI()->GetWindowManager().SendThreadMessage(message);
 }
 
+void CJSONRPCUtils::NotifyItemUpdated(const CSong& song)
+{
+  const CFileItemPtr msgItem(new CFileItem(song));
+  CGUIMessage message(GUI_MSG_NOTIFY_ALL,
+                      CServiceBroker::GetGUI()->GetWindowManager().GetActiveWindow(), 0,
+                      GUI_MSG_UPDATE_ITEM, 0, msgItem);
+  CServiceBroker::GetGUI()->GetWindowManager().SendThreadMessage(message);
```

Comment:
Two calls to  CServiceBroker::GetGUI()->GetWindowManager() in this code block. I would introduce a variable to only have to do this call one time here.